faust/transport/drivers/aiokafka.py

```diff
@@ -487,8 +487,13 @@ def on_generation_id_known(self) -> None:
     def close(self) -> None:
         """Close consumer for graceful shutdown."""
         if self._consumer is not None:
-            self._consumer.set_close()
-            self._consumer._coordinator.set_close()
+            self._consumer._closed = True
+            asyncio.run_coroutine_threadsafe(
+                self._consumer._client.close(), self.app.loop
+            )
+            asyncio.run_coroutine_threadsafe(
+                self._consumer._coordinator.close(), self.app.loop
```

Comment:
Can we avoid accessing the private variable _coordinator?

Reply:
we will add those in aiokafka. They exist in robinhood's version